src/main/java/cz/mzk/scripts/SyncFoxmlFieldsWithSolr.java

```diff
@@ -46,13 +49,15 @@ public static void main(String[] args) throws ParserConfigurationException, Tran
                 // pass FOXML and SolrInputDocument object as parameters
             // if the result SolrInputDocument object has any field to update, send it to Solr
             final String uuidDoc = (String) solrDoc.getFieldValue(SolrField.UUID);
-            final SolrInputDocument solrInputDoc = new SolrInputDocument();
+            final SolrInputDocument solrInputDoc = createSolrInputDocument(solrDoc);
             final Optional<Document> foxml = fedoraClient.getDC(uuidDoc);
 
-            foxml.ifPresent(doc -> fieldSynchronizers.forEach(x -> x.accept(doc, solrInputDoc)));
+            foxml.ifPresentOrElse(doc -> fieldSynchronizers.forEach(x -> x.accept(doc, solrInputDoc)),
+                    () -> missingDocInFedora.writeLine(uuidDoc));
 
-            if (!solrInputDoc.isEmpty()) {
-                solrClient.sendSolrInputDocument(solrInputDoc);
+            if (solrInputDoc.containsKey(SolrField.MODIFIED_DATE)) {
```

Comment:
.isEmpty() nefunguje? nebo proc rozhodujes podle MODIFIED_DATE?

Reply:
Kvoli tomu, ze uz pri vytvarani InputDocumentu obsahuje field name DC_IDENT, cize kontrola isEmpty mi pride v tomto bode zbytocna, lebo tou podmienkou prejde vsetko. 
Preto ma napadlo kontrolovat to podla MODIFIED_DATE, ktory sa pridava len do dokumentov, ktore vykonali nejaku zmenu.
Ci, som to zle pochopil a bude sa to dat kontrolovat aj s pomocou isEmpty?